docs/support.md

```diff
@@ -2,6 +2,22 @@
 
 Here is a page where you can see the OS vendors and versions as well as the browsers we are currently supporting; but also the Python version we are currently using.
 
+## Nuxeo Platform
+
+Nuxeo Drive tends to support all versions currently supported by Nuxeo.
```

Comment:
"tends" implies that it's not 100% sure. I'd just say `"Nuxeo Drive supports all versions [...]"`